### PR TITLE
Added some changes to profile URLs.

### PR DESCRIPTION
**Pull request by sidntrivedi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,3 +18,10 @@ Content to be unblocked from Cyberoam
 - **Blocked Under**: PersonalStorage
 - **Wikipedia**: Amazon Web Services (AWS) is a subsidiary of Amazon.com that provides on-demand cloud computing platforms to individuals, companies and governments, on a paid subscription basis. The technology allows subscribers to have at their disposal a full-fledged virtual cluster of computers, available all the time, through the Internet.
 - **About**: AWS provides a great set of features to developers for hosting and running applications on servers, but AWS is blocked, making the lives of students a tad more difficult.
+
+### 4. A2 Online Judge
+- **URL**: https://a2oj.com/
+- **Blocked under**: Online Judges.
```
Comment (ryzokuken) [inline README.md:24]: Is the actual category called "Online Judges"? I doubt that. The "blocked under" field should include which category it is blocked under.

File: README.md [hunk 2]
```diff
@@ -18,3 +18,10 @@ Content to be unblocked from Cyberoam
...
+- **About**: A2oj is an online judge developed by Ahmed Aly who has been a part of big tech giants like Google,Hackerrank and is now working in Facebook and has been the Chief Judge of Arab ICPC. A2OJ, alongwith many programming challenges provides many other features like programming ladders which really helps and automates the process of sorting the problems for the beginners according to their difficulty and concepts involved. Alongwith this, it also provides a collection of chosen problems from many contests like Arab Collegiate Programming Contest,Codeforces Contests etc. It is trusted and sponsored by Codechef,ACPC,Coach Academy as well as Codability. Thus, Unblocking this from cyberoam would really help the programming aspirants in getting a pre-ordered set of problems for practice and will certainly increase their efficiency.
```
Comment (ryzokuken) [inline README.md:25]: This does not need to be so big, because long "about" heads defeat the purpose. Keep it short and to the point.

**Review by ryzokuken — CHANGES_REQUESTED** [review]
Are you sure it's blocked? It works for me.

screen shot 2018-04-24 at 7 30 36 pm

File: README.md (hunk 1 above)
Comment (sidntrivedi) [reply README.md:24]: Its showing this in my browser.
screenshot from 2018-04-24 19-58-16

File: README.md (hunk 2 above)
Comment (sidntrivedi) [reply README.md:25]: Ok Sir.

File: README.md (hunk 1 above)
Comment (sidntrivedi) [reply README.md:24]: But,if I add it as an exception then it works.So,is it blocked or not?

Comment (ryzokuken) [reply README.md:24]: It's not. Somewhere down the line, some certificate is blocked (probably AWS) and that's why this happens unbanning AWS *should* work.

Comment (sidntrivedi) [reply README.md:24]: Ok Sir.AWS affects a lot of network then..

**Pull request closed without merging**